ArjunaJTS/idl/idlj/pom.xml

```diff
@@ -41,6 +42,239 @@
+                  -->
+                  <arg value="${project.build.directory}/../src/test/java"/>
+                  <arg value="-pkgPrefix"/>
+                  <arg value="TestModule"/>
+                  <arg value="com.hp.mwtests.ts.jts"/>
+                  <arg value="-pkgPrefix"/>
+                  <arg value="CosTransactions"/>
+                  <arg value="org.omg"/>
+                  <arg value="-pkgPrefix"/>
+                  <arg value="orbtests"/>
+                  <arg value="com.hp.mwtests.orbportability.orbspecific"/>
+                  <arg value="-fallTIE"/>
+                  <arg value="-i"/>
+                  <arg value="${project.build.directory}/../../src/main/idl/omg"/>
+                  <arg value="-i"/>
+                  <arg value="${env.JAVA_HOME}/lib"/>
+                  <arg value="-d"/>
+                  <arg value="ORB_CURRENT_SUPPORT=1"/>
+                  <arg value="-fallTIE"/>
+                  <arg value="${project.build.directory}/../../src/test/idl/test.idl"/>
+                  <classpath>
+                    <pathelement location="${env.JAVA_HOME}/lib/tools.jar"/>
+                  </classpath>
+                </java>
+
+                <java classname="com.sun.tools.corba.se.idl.toJavaPortable.Compile">
+                  <arg value="-i"/>
+                  <arg value="${project.build.directory}/../../src/test/idl"/>
+                  <arg value="-td"/>
+                  <!-- TODO figure out why the compiler does not pick up these sources
+                  <arg value="${project.build.directory}/generated-test-sources/idl"/>
+                  -->
+                  <arg value="${project.build.directory}/../src/test/java"/>
+                  <arg value="-pkgPrefix"/>
+                  <arg value="TestModule"/>
+                  <arg value="com.hp.mwtests.ts.jts"/>
+                  <arg value="-pkgPrefix"/>
+                  <arg value="CosTransactions"/>
+                  <arg value="org.omg"/>
+                  <arg value="-pkgPrefix"/>
+                  <arg value="orbtests"/>
+                  <arg value="com.hp.mwtests.orbportability.orbspecific"/>
+                  <arg value="-fallTIE"/>
+                  <arg value="-i"/>
+                  <arg value="${project.build.directory}/../../src/main/idl/omg"/>
+                  <arg value="-i"/>
+                  <arg value="${env.JAVA_HOME}/lib"/>
+                  <arg value="-d"/>
+                  <arg value="ORB_CURRENT_SUPPORT=1"/>
+                  <arg value="-fallTIE"/>
+                  <arg value="${project.build.directory}/../../src/test/idl/TestModule.idl"/>
+                  <classpath>
+                    <pathelement location="${env.JAVA_HOME}/lib/tools.jar"/>
+                  </classpath>
+                </java>
+              </tasks>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+
+      <!-- this one does not seem to work either
```

Comment:
Why doesn't generated-test-sources/idl get added to the maven compiler plugin compileSourceRoots property?